DRAFT_RELEASE_NOTES.md

```diff
@@ -6,7 +6,8 @@ Data access through [GraphQL](https://graphql.org/) has been deprecated and repl
 # Breaking Changes
 _Note: instructions for adapting to these changes are outlined in the upgrade instructions below._
 
- - _There are no breaking changes in teh 1.11 release._
+ - To improve the development cycle and docker build consistency, we have deprecated the docker_build() and local_resources() functions in the Tilt and enable maven docker build for the docker modules. Follow the instruction in the `Finalizing the Upgrade` to avoid duplicated docker image build.
```

Comment:
```suggestion
 - To improve the development cycle and docker build consistency, we have deprecated the `docker_build()` and `local_resources()` functions within Tilt and instead enabled the maven docker build for the docker modules. Follow the instruction in the `Finalizing the Upgrade` to avoid duplicated docker image builds.
```